### DISCUSSION TITLE
It would be great to have the data:delete:bulk command to accept a "hard delete" flag option

### DISCUSSION THREAD

**Discussion by VinceFINET** [forcedotcom/cli#2904] [Ideas]
It would be great to have the data:delete:bulk command to accept a "hard delete" flag option

Like:
sf data delete bulk --hard-delete

By default the option will be false obviously.

You will need to raise an error if the current user does not have the permission to perform such an operation.

It would be awesome as well to have a confirmation to make by default like "are you really sure [yN]?" and read the answer from the keyboard... Of course another flag will be available to enforce the answer to "[Y]es I am sure" so that we can easily script that operation.

Thank you!!!

**Comment by cristiand391** [comment]
tracked internally as [W-15903600
](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001suaz4YAA/view)

**Comment by WillieRuemmele** [comment]
I got a PR up for this : https://github.com/salesforcecli/plugin-data/pull/959 🚀

**Comment by WillieRuemmele** [comment]
this is available in the `latest-rc` as of now, and will be in `latest` on 6/25

**Reply by VinceFINET** [reply]
AWSEOME!!!!
